test/cutting/qpd/test_qpd.py

```diff
@@ -170,7 +181,6 @@ def test_decompose_qpd_instructions(self):
                 e_info.value.args[0]
                 == "Each decomposition must contain either one or two elements. Found a decomposition with (0) elements."
             )
-
```

Comment:
I keep fixing this in my PRs and then reverting it so you dont have to deal with it in code review haha...

I'm just going to leave this change in here and try to sneak it in ;)